scripts/properties2json.js

```diff
@@ -51,24 +51,36 @@ function writeFile(entries) {
   });
 }
 
-function getContentMessages(locale) {
+function readPropertiesFile(filePath) {
   return new Promise(function(resolve, reject) {
-    properties.read(path.join(process.cwd(), config.src, locale, 'messages.properties'), function(message_error, message_properties) {
+    propertiesParser.read(filePath, function(message_error, message_properties) {
       if (message_error && message_error.code !== 'ENOENT') {
         return reject(message_error);
       }
+      resolve(message_properties);
+    });
+  });
+}
 
-      properties.read(path.join(process.cwd(), config.src, locale, 'email.properties'), function(email_error, email_properties) {
-        if (email_error && email_error.code !== 'ENOENT') {
-          return reject(email_error);
-        }
-
+function getContentMessages(locale) {
+  return new Promise(function(resolve, reject) {
+    FS.listTree(path.join(process.cwd(), config.src, locale), function(path, stat) {
+      if (path.indexOf(".properties") !== -1 &&
```

Comment:
https://nodejs.org/dist/latest-v4.x/docs/api/path.html#path_path_extname_p
All you need to do is:
`if(path.extname(filePath) === ".properties")`